### PR TITLE
Creating agenda

### PR DESCRIPTION
**Pull request by navro**

### CODE REVIEW

File: app/assets/stylesheets/partials/_agenda.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.agenda-grid {
+  tr {
+    &.current {
+      td,
+      &.even td,
+      &.odd td {
+        background-color: rgba(252, 152, 0, 0.5);
```
Comment (davidwessman) [inline app/assets/stylesheets/partials/_agenda.scss]: Perhaps we should add `$orange-highlight = rgba(252, 152, 0, 0.5);` to 
https://github.com/fsek/voting/blob/master/app/assets/stylesheets/colors.scss

File: app/models/agenda.rb [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    end
+  end
+
+  def current?
+    status == CURRENT
+  end
+
+  private
+
+  def parent_validation
```
Comment (davidwessman) [inline app/models/agenda.rb]: Would like some test for this, but it seems valid 😄

File: app/views/admin/adjustments/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<%= simple_form_for [:admin, @adjustment] do |f| %>
+  <%= f.input :agenda_id, collection: Agenda.all, include_hidden: false,
+                include_blank: false, input_html: { class: 'select2' } %>
+  <%= f.input :presence, include_blank: false, collection: adjustment_collection %>
+  <%= f.hidden_field :user_id, value: @adjustment.user_id %>
+  <div class="table">
+    <%= f.button :submit, class: 'btn primary' %>
```
Comment (davidwessman) [inline app/views/admin/adjustments/_form.html.erb]: `class: 'btn primary'` is not needed here. Added through `SimpleForm` setup.

File: app/views/admin/adjustments/edit.html.erb [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<div class="col-md-6 col-md-offset-3 col-sm-12 reg-page">
+  <div class="headline">
+    <h1><%= t('adjustment.edit') %></h1>
+  </div>
+  <%= render 'form' %>
+    <%= link_to t(:destroy_attr, attr: model_name(Adjustment)),
```
Comment (davidwessman) [inline app/views/admin/adjustments/edit.html.erb]: I tried to use these translations and I think it ended up rather messy.
I do prefer to use `t('adjustment.destroy')`.

File: app/views/layouts/_admin_dropdown.html.erb [hunk 5]
```diff
@@ -48,5 +48,10 @@
         <%= link_to t('user.vote_administration'), admin_vote_users_path %>
       </li>
     <% end %>
+    <% if can? :manage, Agenda %>
```
Comment (davidwessman) [inline app/views/layouts/_admin_dropdown.html.erb]: `:modify`

**Comment by davidwessman** [comment]
Gonna test it out locally as well, looking really good 😄

**Comment by davidwessman** [comment]
The buttons on the admin pages are great, would be good to have one for the Agenda as well 😄 
Other than my comments, +1

**Comment by navro** [comment]
> The buttons on the admin pages are great, would be good to have one for the Agenda as well :smile:
> Other than my comments, +1

I fixed all your comments except the authorizations. Let's wait until we decide what to do in `web`. I also added some tests for `Agenda` :smile:

File: app/controllers/admin/adjustments_controller.rb [hunk 6]
```diff
@@ -0,0 +1,49 @@
+class Admin::AdjustmentsController < ApplicationController
+  load_permissions_and_authorize_resource
```
Comment (davidwessman) [inline app/controllers/admin/adjustments_controller.rb]: This controller is only interacted by admins, so we could remove the `before_action`

**Comment by davidwessman** [comment]
:+1: 
I like the admin namespace.
If we have a *normal* controller as well I think we should stick with `authorize!(:manage, Model)` until we create an `AdminAbility`. 

I think we should get this merged (since we want to connect a lot to it) and then fix the authorization through the whole project later.

**Comment by navro** [comment]
> :+1:
> I like the admin namespace.
> If we have a normal controller as well I think we should stick with authorize!(:manage, Model) until we create an AdminAbility.
> 
> I think we should get this merged (since we want to connect a lot to it) and then fix the authorization through the whole project later.

I agree :)

**Pull request merged**